Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -220,24 +231,25 @@ The PHP attribute :php:`\TYPO3\CMS\Core\Attribute\AsEventListener` supports the
 following properties (which are all optional):
 
 :php:`identifier`
-    A unique identifier must be declared which identifies the event listener,
-    and orderings can be build upon the identifier. It this property is not
-    explicitly defined, the service name is used instead.
+    A unique identifier should be declared which identifies the event listener,
+    and orderings can be build upon the identifier. If this property is not
+    explicitly defined, the service name is automatically used instead.
 
 :php:`before`
     This property allows a custom sorting of registered listeners. The listener
     is then dispatched before the given listener. The value is the identifier of
-    another event listeners.
+    another event listener.
 
 :php:`after`
     This property allows a custom sorting of registered listeners. The listener
     is then dispatched after the given listener. The value is the identifier of
-    another event listeners.
+    another event listener.
 
 :php:`event`
-    The fully-qualified class name (FQCN) of the event to be dispatched. This
-    property can be omitted, if the FQCN is used as type declaration of the
-    argument of the dispatched method.
+    The fully-qualified class name (FQCN) of the dispatched event, that the
+    listener wants to react to. It is recommended to not specify this property,
+    but to use the FQCN as type declaration of the argument within the dispatched method
+    (usually :php:`__invoke(EventName $event) ` unless overridden with a custom :php:`method`).
```

Comment:
I don't understand this: "unless overridden with a custom method" in combination with the type declaration. Additionally, method is not a PHP method, just a word, so no need to use the php role.

Reply:
It means, you can define your own method other than "__invoke". But I agree we could simplify this sentence.

```suggestion
    (usually :php:`__invoke(EventName $event)`).
```